frontend/packages/schema-editor/src/SchemaEditorApp.test.tsx

```diff
@@ -1,103 +1,37 @@
 import React from 'react';
 import { act, render as rtlRender, screen } from '@testing-library/react';
 import { SchemaEditorApp } from './SchemaEditorApp';
-import { dataMock } from '@altinn/schema-editor/mockData';
 import { jsonMetadataMock } from 'app-shared/mocks/datamodelMetadataMocks';
+import { jsonSchemaMock } from '../test/mocks/jsonSchemaMock';
 import userEvent from '@testing-library/user-event';
-import { useSchemaEditorAppContext } from './hooks/useSchemaEditorAppContext';
-import { uiSchemaNodesMock } from '../test/mocks/uiSchemaMock';
-import { AUTOSAVE_DEBOUNCE_INTERVAL } from 'app-shared/constants';
-import { buildJsonSchema, buildUiSchema, SchemaModel } from '@altinn/schema-model';
+import { textMock } from '../../../testing/mocks/i18nMock';
 
-jest.useFakeTimers({ advanceTimers: true });
+const user = userEvent.setup();
```

Comment:
I think we should move this to the tests where ```user``` is used, as [it is recommended by the documentation provided by the testing library.](https://testing-library.com/docs/user-event/intro/#writing-tests-with-userevent)

Reply:
It should not be a problem here, but of course it doesn't hurt. I'll fix.